README.md

```diff
@@ -1 +1,3 @@
-# TE-sync-git-lesson
\ No newline at end of file
+# TE-sync-git-lesson
+
+I MADE A CHANGE
```

Comment:
This seems like it could be more efficient.

Reply:
ur wrong this compiles on my machine